src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let finalPrice = 0;
+
+  if (days >= 7) {
+    finalPrice = 40 * days - 50;
```

Comment:
Replace the hardcoded value '40' with a constant that represents the daily cost. Also, replace '50' with a constant representing the long-term discount.